components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/StatsUpdateUtil.java

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Shall we merge this code into APIProviderImpl rather than having a whole new class for it. Since the code in this class gets triggered by Publisher App, better to put it inside APIProviderImpl.